BLSQ/iaso
# WC2-471: Child exit by death not showing in dashboard

Explain what problem this PR is resolving

Related JIRA tickets : [WC2-471](https://bluesquare.atlassian.net/browse/WC2-471), [WC2-472](https://bluesquare.atlassian.net/browse/WC2-472), [WC2-473](https://bluesquare.atlassian.net/browse/WC2-473)

## Self proofreading checklist



## Changes

Some exit types(dismissed, death and defaulter) was not calculated as expected. Refactored journey mapper function to fix the exit types!

## How to test
From the local instance with wfp coda database, launch with `docker-compose run iaso manage wfp_etl` the ETL script in order to generate data for tableau dashboard. Then check some exity type:
 - defaulter: [http://localhost:8081/wfp/debug/254/](http://localhost:8081/wfp/debug/254/)
 - death: [http://localhost:8081/wfp/debug/431/](http://localhost:8081/wfp/debug/431/) and http://localhost:8081/wfp/debug/315/
 - dismissal for cheating: [http://localhost:8081/wfp/debug/430/](http://localhost:8081/wfp/debug/430/)

## Print screen / video

[Administration-de-Iaso-Iaso.webm](https://github.com/user-attachments/assets/407b3de2-062d-4755-a551-bb3166491302)



[WC2-471]: https://bluesquare.atlassian.net/browse/WC2-471?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WC2-472]: https://bluesquare.atlassian.net/browse/WC2-472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WC2-473]: https://bluesquare.atlassian.net/browse/WC2-473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
